### PR TITLE
8289471: Issue in Initialization of keys in ErrorMsg.java and XPATHErrorResources.java

### PR DESCRIPTION
**Pull request by shruacha1234**
OpenJDK PR : https://github.com/openjdk/jdk/pull/9369
OpenJDK bug : https://bugs.openjdk.org/browse/JDK-8289471

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8289471](https://bugs.openjdk.org/browse/JDK-8289471): Issue in Initialization of keys in ErrorMsg.java and XPATHErrorResources.java

### Reviewers
 * [Paul Hohensee](https://openjdk.org/census#phh) (@phohensee - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk11u-dev pull/1273/head:pull/1273` \
`$ git checkout pull/1273`

Update a local copy of the PR: \
`$ git checkout pull/1273` \
`$ git pull https://git.openjdk.org/jdk11u-dev pull/1273/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1273`

View PR using the GUI difftool: \
`$ git pr show -t 1273`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk11u-dev/pull/1273.diff">https://git.openjdk.org/jdk11u-dev/pull/1273.diff</a>

</details>

### CODE REVIEW

**Comment by shruacha1234** [comment]
@RealCLanger Can you please review this PR

**Review by phohensee — APPROVED** [review]
Looks clean, lgtm.

**Comment by RealCLanger** [comment]
Same here: Before integration, this needs maintainer approval. See point 6 of the [Wiki](https://wiki.openjdk.org/display/JDKUpdates/How+to+contribute+a+fix). @phohensee ?

**Comment by phohensee** [comment]
Tagged and commented on the JBS issue.

**Pull request closed without merging**
